### PR TITLE
doc/cephadm: adopt: fix misleading `apply` examples

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Fixes: https://tracker.ceph.com/issues/46052
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephadm/adoption.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ Adoption process
    each file system.  You can list file systems by name with ``ceph fs
    ls``.  For each file system::
 
-     # ceph orch apply mds <fs-name> <num-daemons>
+     # ceph orch apply mds <fs-name> [--placement <placement>]
```
Comment (varshar16) [inline doc/cephadm/adoption.rst:116]:
```suggestion
     # ceph orch apply mds <fs-name> [--placement=<placement>]
```

**Comment by Devp00l** [comment]
jenkins test docs

**Comment by sebastian-philipp** [comment]
jenkins test docs

**Comment by sebastian-philipp** [comment]
jenkins test docs

**Comment by sebastian-philipp** [comment]
jenkins test make check

**Comment by sebastian-philipp** [comment]
jenkins test docs

**Comment by sebastian-philipp** [comment]
jenkins test docs

**Pull request merged**